lib/assets/models/index.js

```diff
@@ -17,7 +17,7 @@ if (config.use_env_variable) {
 fs
   .readdirSync(__dirname)
   .filter(function(file) {
-    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js' || file.slice(-7) === '.coffee');
   })
   .forEach(function(file) {
     var model = sequelize['import'](path.join(__dirname, file));
```

Comment:
Sorry I am new to sequelize and did not follow your comment. Happy to fix with further clarification